azurelinuxagent/ga/collect_telemetry_events.py

```diff
@@ -459,6 +462,7 @@ def process_events(self):
                     logger.verbose("Processing event file: {0}", event_file_path)
 
                     event = self._read_and_parse_event_file(event_file_path)
+                    redact_event_msg(event)
```

Comment:
how is this cleanup related to the one at line 363?

Reply:
We have two separate collections, one for agent events and one for extension events.

Line 363 for extension events and 465 for agent events. Even though we cleanup the agent events before we write to disk, we still need here just in case if old daemons include sas tokens in the events.

Reply:
ah, yes, "legacy" events. makes sense